packages/wrangler/src/autoconfig/run.ts

```diff
@@ -120,3 +168,99 @@ export async function runAutoConfig(
 
 	return;
 }
+
+function usesTypescript(projectPath: string) {
+	return hasTsConfig(projectPath);
+}
+
+function hasTsConfig(path: string) {
+	return existsSync(join(`${path}`, `tsconfig.json`));
+}
+
+type PackageToInstall = {
+	package: string;
+	depType: "dependency" | "devDependency";
+};
+
+type ScriptToAdd = {
+	name: string;
+	value: string;
+};
+
+export async function buildAndConfirmOperationsSummary(
+	autoConfigDetails: AutoConfigDetails,
+	wranglerConfigToWrite: RawConfig
+): Promise<{
+	confirmed: boolean;
+	packagesToInstall: PackageToInstall[];
+	scriptsToAdd: ScriptToAdd[];
+}> {
+	const packagesToInstall: PackageToInstall[] = [];
+	const scriptsToAdd: ScriptToAdd[] = [];
+	if (autoConfigDetails.packageJson) {
+		const devDependencies = autoConfigDetails.packageJson.devDependencies ?? {};
+		const shouldInstallLatestWrangler =
+			!("wrangler" in devDependencies) ||
+			(typeof devDependencies["wrangler"] === "string" &&
+				!devDependencies["wrangler"].startsWith("^"));
+		if (shouldInstallLatestWrangler) {
+			packagesToInstall.push({ package: "wrangler", depType: "devDependency" });
+		}
+
+		const isFullstackFramework = false; // TODO: handle this logic appropriately
```

Comment:
What will this mean here?

Reply:
Basically phase 2, checking the framework in use and its configuration to understand if the application is a full stack / SSR one

Reply:
@dario-piotrowicz could you expand on what we'll be doing here? My understanding was that this would mostly/entirely be handled by the `configure()` callbacks